typings/internalBinding/url_pattern.d.ts

```diff
@@ -0,0 +1,20 @@
+export class URLPattern {
+  protocol: string
+  username: string
+  password: string
+  hostname: string
+  port: string
+  pathname: string
+  search: string
+  hash: string
+
+  constructor(input: Record<string, string> | string, options?: { ignoreCase: boolean });
+  constructor(input: Record<string, string> | string, baseUrl?: string, options?: { ignoreCase: boolean });
```

Comment:
```suggestion
  constructor(input: Record<string, string> | string, options?: { ignoreCase?: boolean });
  constructor(input: Record<string, string> | string, baseUrl?: string, options?: { ignoreCase?: boolean });
```
since an empty options object is still permitted